history.txt

```diff
@@ -68,7 +68,7 @@ Developer updates include:
 
 
 This release also upgrades the version of Bio-Formats which OMERO uses to
-`5.5.2 <https://www.openmicroscopy.org/site/support/bio-formats5.5/about/whats-new.html#june-15>`_.
+`5.5.2 <ttps://www.openmicroscopy.org/2017/06/15/bio-formats-5-5-2.html>`_.
```

Comment:
Typo missed here, I'll fix in a following PR as there's at least one plone link I found when reviewing the docs